subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -176,7 +177,17 @@ static int cmd_read(const struct shell *sh, size_t argc, char **argv)
 	if (argc > 2) { /* read + path + options(data type) */
 		dtype = argv[2];
 	}
-	if (strcmp(dtype, "-s") == 0) {
+	if (strcmp(dtype, "-x") == 0) {
+		const char *buff;
+		uint16_t buff_len = 0;
+
+		ret = lwm2m_engine_get_res_buf(pathstr, (void **)&buff,
```

Comment:
This function is now called `lwm2m_get_res_buf()`

Reply:
Got it. I should not use my zephyr 3.2 patches blindly.